bigframes/operations/__init__.py

```diff
@@ -60,6 +60,11 @@ def order_preserving(self) -> bool:
         """Whether the row operation preserves total ordering. Can be pruned from ordering expressions."""
         return False
 
+    @property
+    def pruning_compatible(self) -> bool:
+        """Whether the operation preserves locality o"""
```

Comment:
I see a hanging "o". Was that meant to be `"or ..."`?

Reply:
I'd also like some more information for help in determining when an operation would be pruning compatible.

Reply:
Actually didn't end up using this. Removed from new revision. Later on will add some concept of "inverse" operation to help normalize predicates. For now though, only considering range and equality predicates between column and a constant.